pkg/operator/ceph/controller/spec.go

```diff
@@ -686,12 +690,20 @@ func PrivilegedContext(runAsRoot bool) *v1.SecurityContext {
 func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Container {
 
 	var maxLogSize resource.Quantity
+	rotation := "7"
 	if c.LogCollector.MaxLogSize != nil {
 		size := c.LogCollector.MaxLogSize.Value() / 1000 / 1000
 		if size == 0 {
 			size = 1
 			logger.Info("maxLogSize is 0M setting to minimum of 1M")
 		}
+
+		if strings.Contains(daemonID, "*-client.rbd-mirror*") {
+			rotation = "28"
```

Comment:
Since `rotate` is adjusted instead of `maxsize` now, this probably doesn't belong in `c.LogCollector.MaxLogSize != nil` branch.

Reply:
removed, I'll test again and update the output.